m2cgen/interpreters/interpreter.py

```diff
@@ -1,18 +1,67 @@
 import re
+import sys
 
 from m2cgen import ast
 
 
-class BaseInterpreter:
+class BaseAstInterpreter:
```

Comment:
I really don't like `Ast` prefix here. It's kind of obvious what's being interpreted. Same true for `AstToCode...` why not just `CodeGenInterpreter`? Or even just `ToCodeInterpreter`?